mapping.go

```diff
@@ -117,5 +115,6 @@ func (m *mapping) ExternalAddr() (net.Addr, error) {
 }
 
 func (m *mapping) Close() error {
-	return m.proc.Close()
+	m.teardown(m)
```

Comment:
Can we just directly call `m.nat.removeMapping(m)`?

Reply:
Yeah, and that lets us get rid of the `teardown` member variable.